AutoMad/autoMad.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 22 13:14:59 2024
+
+@author: dragon
+"""
+
+import random
+import re
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+import nltk
+
+import madDictionary
+import madLibFeeder
+import wordGrabber
+
+def posToCategoryOld(pos):
+    if pos in NOUNS:
+        return 'noun'
+    if pos in VERBS:
+        return 'verb'
+    if pos in CONJUNCTIONS:
+        return 'conjunction'
+    if pos in NUMBER:
+        return 'number'
+    if pos in ADJECTIVES:
+        return 'adjective'
+    if pos in ADVERBS:
+        return 'adverb'
+    if pos in SILLIES:
+        return 'sillyword'
+    if pos in UNKNOWN:
+        return 'sillyword'
+    if pos in PREPOSITIONS:
+        return 'preposition'
+    if pos in PRONOUNS:
+        return 'pronoun'
+    if pos in ARTICLES:
+        return 'article'
+    if pos in NAMES:
+        return 'name'
+    return pos + "word Not Found"
+
+NOUNS = ["NN","NNS"]
+VERBS = ["EX","MD","VB","VBD","VBG","VBN","VBP","VBZ"]
+CONJUNCTIONS =["CC"]
+NUMBER = ["CD","LS"]
+ADJECTIVES = ["JJ","JJR","JJS","WDT"]
+ADVERBS = ["RB","RBR","RBS","RP","WRB"]
+SILLIES = ["UH"]
+UNKNOWN = ["FW","PDT","POS"]
+PREPOSITIONS = ["IN","TO","WP","WP$"]
+PRONOUNS = ["PRP","PRP$"]
+ARTICLES = ["DT"]
+NAMES = ["NNP","NNPS"]
```

Comment:
I think you can delete a bunch of this now?

Reply:
yes